tests/unit-tests/UnitTests/Services/SubmitServiceTests.cs

```diff
@@ -38,7 +45,15 @@ public SubmitServiceTests()
                     FakePaymentSubmission = false
                 });
 
-            _service = new SubmitService(_mockGateway.Object, _mockPageHelper.Object, _mockEnvironment.Object, _mockIOptions.Object);
+            _mockSchemaFactory.Setup(_ => _.Build(It.IsAny<string>()))
+                .ReturnsAsync(new FormSchema {
+                    GenerateReferenceNumber = false
+                });
```

Comment:
We need to be consistent in this file with if our .Setup is chained on the same line or on the line below, we swap between these throughout this file

Reply:
Moved all to the line below.